src/20231205_Multi_IncreaseSupplyAndBorrowCapsAt100UtilizationDecember2023/IncreaseSupplyAndBorrowCapsAt100UtilizationDecember2023.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Increase Supply and Borrow Caps at 100% Utilization - December 2023"
+author: "Alice Rozengarden (@Rozengarden - Aave-chan initiative)"
+discussions: "https://governance.aave.com/t/arfc-increase-supply-and-borrow-caps-at-100-utilization-december-2023/15754"
+---
+
+## Simple Summary
+
+This ARFC proposes to increase supply and borrow caps on assets that currently have 100% or close to 100% utilization.
+
+## Motivation
+
+The ARFC proposes increasing supply and borrow caps on a number of assets to meet the ongoing market demand for depositing and borrowing. This increase will enable new Aave users to join Aave V3 and allow existing users to expand their positions. We will use the caps update framework and institute these changes as a direct-to-AIP proposal after review from risk managers. Supply caps on assets at supply cap will be increased by 50%. Borrow caps on 100% borrow utilization assets will be increased by 50%.
```

Comment:
this text is a bit nonsense when in the end you change a single cap of a single asset

Reply:
pushed a new text, does it seem better?